test-support/lit-exec-qemu.py

```diff
@@ -7,30 +7,34 @@
 # arguments as llvm-project/libcxx/utils/run.py.
```

Comment:
Please format this file with `black -l79` (TODO: automate this check)

Reply:
Done